test/test_examples_3d.jl

```diff
@@ -103,6 +109,13 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
       linf = [0.0016329580288680923, 0.0029870270738030775, 0.009177053066089513, 0.009177053066084184])
   end
 
+  @testset "taal-confirmed elixir_hyp_diff_llf.jl with initial_refinement_level=2, surface_flux=Trixi.flux_upwind)" begin
+    test_trixi_run(joinpath(EXAMPLES_DIR, "elixir_hyp_diff_llf.jl"),
+      l2   = [0.0015377708559180534, 0.011376842329542572, 0.011376842329542624, 0.0113768423295426],
+      linf = [0.02271542063004106, 0.10191067906109286, 0.10191067906109552, 0.10191067906109286],
+      initial_refinement_level=2, surface_flux=Trixi.flux_upwind)
```

Comment:
This flux should be exported (it is on `dev`).

Reply:
Fixed in b3520bd.